core/src/models/Property.ts

```diff
@@ -41,19 +39,19 @@ import { RecordProperty } from "./RecordProperty";
 import { Run } from "./Run";
 import { Source } from "./Source";
 
-export function propertyJSToSQLType(jsType: string) {
-  const map = {
-    boolean: config.sequelize.dialect === "sqlite" ? "text" : "boolean", // there is no boolean type in SQLite
-    date: "bigint", // we store things via timestamps in the DB
-    email: "text",
-    float: "float",
-    integer: "bigint",
-    phoneNumber: "text",
-    string: "text",
-    url: "text",
-  };
-
-  return map[jsType];
+const jsMap = {
+  boolean: config.sequelize.dialect === "sqlite" ? "text" : "boolean", // there is no boolean type in SQLite
+  date: "bigint", // we store things via timestamps in the DB
+  email: "text",
+  float: "float",
+  integer: "bigint",
+  phoneNumber: "text",
+  string: "text",
+  url: "text",
+} as const;
+
+export function propertyJSToSQLType(jsType: keyof typeof jsMap) {
+  return jsMap[jsType];
 }
```

Comment:
Nitpick/stylistic
This could be single-lined `const propertyJSToSQLType = (jsType: keyof typeof jsMap) => jsMap[jsType];`